README.md

```diff
@@ -1,27 +1,20 @@
 # kube-ao
+
 Kubernetes assets for running AppOptics
 
 ## About
 
 This repo contains two Kubernetes assets:
-- Deployment - A single pod to run on a master node and talk to the kube-api. Communication with kube-api is facilitated by the included ServiceAccount with RBAC restrictions.
-- DaemonSet - A DaemonSet that runs a pod on every node in your cluster and publishes HostAgent and Docker metrics.
+- Deployment - A single pod to run on a master node and talk to the Kubernetes API to send Kubernetes specific metrics to AppOptics.
```

Comment:
does it have to run one a master node ? I mean all we need is to be able to talk to the k8s API right ?

Reply:
You're correct, this is a relic of running the agent as a package on the node itself where it would talk to the kube-api over localhost. I'll remove this.